README.md

```diff
@@ -157,13 +157,13 @@ datadreamer --config <path-to-config>
 
 ### 🔧 Additional Parameters
 
-- `--task`: Choose between detection and classification. Default is `detection`.
+- `--task`: Choose between detection, classification, instance segmentation and semantic segmentation. Default is `detection`.
```

Comment:
Should be:
 Choose between detection, classification, instance-segmentation. Default is `detection`.

Reply:
Thanks, missed that!